### DISCUSSION TITLE
Display number of elements in array

### DISCUSSION THREAD

**Discussion by hollandThomas** [Code-Forge-Net/Remix-Dev-Tools#20] [Ideas]
Hey, great work with this project!

Similar to Chrome console, where the number of elements in an array are displayed like this `(10)`
![Bildschirmfoto 2023-07-18 um 11 29 32](https://github.com/Code-Forge-Net/Remix-Dev-Tools/assets/20690465/eab910c8-eab8-4d19-8fa4-6fa828c5ef6d)

it would be nice to see something similar for route loader data somewhere in this UI
![Bildschirmfoto 2023-07-18 um 11 28 55](https://github.com/Code-Forge-Net/Remix-Dev-Tools/assets/20690465/3f009ff8-8016-4732-bba8-04aaeea37a0c)

**Comment by AlemTuzlak** [comment]
Hi @hollandThomas thank you! Both the issues you submitted are great ideas, I will look into them because I am using a third party package to show the data if the package itself would allow for this. If yes this will be added soon, if not I would have to look into other options!

**Reply by AlemTuzlak** [reply]
@hollandThomas good news, this will work in the upcoming release

**Comment by AlemTuzlak** [comment]
implemented
